ultravox/tools/ds_tool/ds_tool.py

```diff
@@ -210,6 +211,12 @@ class DatasetToolArgs:
     private: bool = simple_parsing.field(default=False)
     token: Optional[str] = None
 
+    # Chunk processing parameters
+    max_dynamic_chunks: int = simple_parsing.field(default=10)
+    min_samples_per_chunk: int = simple_parsing.field(
+        default=10000
```

Comment:
I'm choosing sample number instead of sample size because our processing failures are usually correlated with # of sample not sample size.